tests/tests.src.js

```diff
@@ -6620,6 +6620,11 @@
 			Error,
 			'Parse error: forbidden code point when `allowUnsafeSymbols: true` and `strict: true`'
 		);
+		equal(
+			he.encode('äöüÄÖÜ', { 'decimalOutput': true}),
+			'&#228;&#246;&#252;&#196;&#214;&#220;',
+			'encode to decimal HTML entities'
+		);
```

Comment:
Let’s add some more tests that combine `decimal: true` with other options, such as `useNamedReferences`.